lib/delta_filters.rb

```diff
@@ -0,0 +1,23 @@
+module Inbox
+  class DeltaFilters
```

Comment:
I thought about that, but really, we shouldn't be building URLs nor QueryStrings by hand. I was hoping to change the internals to leverage either `URI::Generic` (and friends) or `Addressable::URI` to do the actual URI building. Then this object's job would be purely about building the list of filters.